### PR TITLE
Issue #9348: Extract common bash CI functions into util.sh

### PR DESCRIPTION
**Pull request by aryaniiit002**
For Issue #9348

### CODE REVIEW

**Comment by romani** [comment]
Please fix CI

File: .ci/travis/travis.sh [hunk 1]
```diff
@@ -2,31 +2,7 @@
...
-function getCheckstylePomVersion {
-  echo "$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
-                      -Dexec.args='${project.version}' \
-                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)"
-}
-
-removeFolderWithProtectedFiles() {
-  find $1 -delete
-}
+source ./.ci/util.sh
```
Comment (aryaniiit002) [inline .ci/travis/travis.sh:5]: what's the correct way for this?

File: .ci/no-exception-test.sh [hunk 2]
```diff
@@ -1,31 +1,7 @@
...
-    echo "Target project $PROJECT is already cloned, latest changes will be fetched"
-    cd $PROJECT
-    git fetch
-    cd ../
-  else
-    for i in 1 2 3 4 5; do git clone $CLONE_URL && break || sleep 15; done
-  fi
-  cd ../
-}
+source ./util.sh
```
Comment (AkMo3) [inline .ci/no-exception-test.sh:4]: This should be `source ./.ci/util.sh`

**Review by AkMo3 — CHANGES_REQUESTED** [review]

File: .ci/travis/travis.sh [hunk 3]
```diff
@@ -2,31 +2,7 @@
...
```
Comment (aryaniiit002) [inline .ci/travis/travis.sh:6]: what's the correct way for this?

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Points:

File: .ci/no-exception-test.sh (hunk 2 above)
Comment (aryaniiit002) [reply .ci/no-exception-test.sh:4]: and for travis.sh, what's the correct way?

Comment (AkMo3) [reply .ci/no-exception-test.sh:4]: Same, this will work for travis also. You can run locally to check changes. `./.ci/travis/travis.sh test` .

Comment (aryaniiit002) [reply .ci/no-exception-test.sh]: done

Comment (AkMo3) [reply .ci/no-exception-test.sh]: Resolved.

**Comment by strkkk** [comment]
@aryaniiit002 do you have windows env? To resolve ci issue you can recreate this file in any linux env. I had such issue, windows creates sh scripts with incorrect mode and I couldnt find a good way to resolve it on windows and had to use linux instead

**Comment by aryaniiit002** [comment]
i am using linux.

**Comment by aryaniiit002** [comment]
```
aryaniiit002@ENIAC:~/Desktop/checkstyle$ git diff
diff --git a/.ci/util.sh b/.ci/util.sh
old mode 100644
new mode 100755

```

**Review by romani — APPROVED** [review]

**Review by strkkk — APPROVED** [review]

**Pull request merged**